src/AWSS3.jl

```diff
@@ -13,6 +13,8 @@ export S3Path,
     s3_put,
     s3_get,
     s3_get_file,
+    s3_exists_versioned,
+    s3_exists_unversioned,
```

Comment:
I think it might be good to keep these internal as implementation details behind `s3_exists`